packages/nextjs-contentful-renderer/src/utils/getPageId/getPageId.ts

```diff
@@ -1,6 +1,15 @@
-import { Route, RouteVariant } from '../contentful/getRouteBySlug/types';
+import { Replica, Route, RouteVariant } from '../contentful/getRouteBySlug/types';
+
+export const getPageId = (
+  replicaRoute: Replica | Route,
+  campaign: undefined | string | string[] = undefined
+): string => {
+  if (Object.keys(replicaRoute).includes('page')) {
```

Comment:
there is a more idiomatic way of doing this check, see [type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) in the documentation